Editor/BootstrapperConfig.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+
+namespace OneSignalPush.Bootstrapper
+{
+    static class BootstrapperConfig
+    {
+
+        public static readonly string OneSignalScope = "com.onesignal";
```

Comment:
Let's use `com.onesignal.unity` for the scope so if OneSignal adds non-unity packages at this scope they won't show up in unity.